cartridge/graphql.lua

```diff
@@ -135,6 +130,7 @@ local function add_callback(opts)
             description = opts.doc,
         }
     end
+    vars.graphql_schema = nil
```

Comment:
How about adding the same to `add_callback_prefix` and `add_mutation_prefix`?

Reply:
May be it could improve error message in case if we try to use some "prefix" and there are no any callbacks/mutations (to be honest it's strange case). Fixed.